101-create-backup-vault-policy/azuredeploy.json

```diff
@@ -0,0 +1,180 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the Vault"
+            }
+        },
+        "location": {
+            "type": "string",
+            "allowedValues": [
+                "East US",
+                "East US 2",
+                "West US",
+                "Central US",
+                "North Central US",
+                "South Central US",
+                "North Europe",
+                "West Europe",
+                "East Asia",
+                "South East Asia",
+                "East China",
+                "West Japan",
+                "Australia East",
+                "Australia Southeast",
+                "Brazil South",
+                "Japan East",
+                "Japan West"
+            ],
+            "metadata": {
+                "description": "This is the location where the backup vault will be created"
+            }
+        },
+        "skuName": {
+            "type": "string",
+            "defaultValue": "Standard",
+            "allowedValues": [
+                "Standard",
+                "Premium"
+            ],
+            "metadata": {
+                "description": "SKU for the vault"
+            }
+        },
+        "policyName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the Backup Policy"
+            }
+        },
+        "workloadType": {
+            "type": "string",
+            "allowedValues": [
+                "IaasVM"
+            ],
+            "metadata": {
+                "description": "Type of Workload. Currently Supported only IaasVM."
+            }
+        },
+        "backupType": {
+            "type": "string",
+            "allowedValues": [
```

Comment:
This could become a variable since there is only one possible value here and user input doesn't really change anything